templates/_blockonomics_error_helper.php

```diff
@@ -0,0 +1,41 @@
+<?php
+$blockonomics = new Blockonomics;
+
+$error_type = get_query_var('blockonomics_error_type', NULL);
+$error_msg = get_query_var('blockonomics_error_msg', NULL);
+
+if (isset($order) && $order['status'] == -2) {
+  // Partial payment is recevied
+  $error_type = 'paid_amount_is_less';
+}
+
+$error_title = NULL;
+$has_error = TRUE;
+
+if ($error_type == 'paid_amount_is_less') {
```

Comment:
Such complex logic is not required anymore. As I mentioned in #279 we just now need one error message for everything